SCClassLibrary/Common/GUI/PlusGUI/Core/HistoryGui.sc

```diff
@@ -167,25 +147,20 @@ HistoryGui : JITGui {
 		.action_({ |lview|
 			var index = lview.value;
 			if (lview.items.isEmpty) {
-				"no entries yet.".postln;
+				// 	"no entries yet.".postln;
```

Comment:
mixed whitespace after this comment

Reply:
actually the commented-out post can just be removed, so I did that.